tools/ci_build/get_docker_image.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import argparse
+import collections
+import hashlib
+import os
+import shlex
+import sys
+from logger import get_logger
+
```

Comment:
Would you please add some comments about how the hash was created ?
And, how to force regenerating the docker images when needed?

Reply:
Added some info in the ArgumentParser description.
This script only pulls from or pushes to the container registry. To regenerate, we can delete the image from the container registry.

---
In reply to: [523794764](https://github.com/microsoft/onnxruntime/pull/5811#discussion_r523794764) [](ancestors = 523794764)